changelog/unreleased/fix-copy-responce-webdav.md

```diff
@@ -0,0 +1,6 @@
+Bugfix: Fixed the response code when copying the shared from to personal
+
+We fixed the response code when copying the file from shares to personal space with a secure view role.
```

Comment:
```suggestion
Bugfix: Fixed the response code when copying from a share to a personal space

We fixed the response code when copying a file from a share to a personal space with a secure view role.
```

Maybe this says it more clearly?

Reply:
OMG Thank you @phil-davis With the `Bugfix` descriptions it's happened because of the annoying validation saying `too long`  `still too long`